src/server/routes/selectData.js

```diff
@@ -1,10 +1,7 @@
 import monthData from './monthData';
 import annualData from './annualData';
-import appConfig from '../../../appConfig';
```

Comment:
I think we will need `appConfig`. It is used at Line 18 and 21.

Reply:
Yup -- will add back -- hasty merge.

Reply:
Added back. See below.